src/System.Private.CoreLib/shared/System/Runtime/ExceptionServices/ExceptionDispatchInfo.cs

```diff
@@ -2,6 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable enable
 using System.Diagnostics;
 
 namespace System.Runtime.ExceptionServices
```

Comment:
Again, seems like parameter to ctor here are nullable. (In the sense that the code does not mind getting nulls)

Reply:
> Again, seems like parameter to ctor here are nullable

It's private and is only called by Capture which has already validated that it's not null.  It should stay non-nullable.

Reply:
Correct, that is why I didn't make it nullable.